### ISSUE TITLE
Interactivity for output filtering

### ISSUE THREAD

**Issue by pkosiec** [kubeshop/botkube#788]
## Description

This is an extension of the task https://github.com/kubeshop/botkube/issues/742 that brings Slack interactivity for the new feature.

## AC

- Implement the design according to the prototypes: https://www.figma.com/proto/UcaGQ1UiBziOQbNPmSwE0T/grep?page-id=0%3A1&node-id=2%3A173&viewport=163%2C220%2C0.08&scaling=scale-down&starting-point-node-id=2%3A173&show-proto-sidebar=1
    - Support just the first flow 
- Slack only for now
- Manual tests of the interactivity part for now
- Skip "Open in editor" button
- Dropped: ~Stretch: Support AND/OR dropdown + additional text fields rendered as dynamic form (you can add them as many as you want).~
- Minimal scope: static form: 
   - single text field
   - To agree with @lpetkov - whether this text field is in modal or embedded directly in the message
        - Always attach "Filter output" button for each command output
        - Single modal open by button with a single text filter
- Filtering logic is out of scope as it is implemented as a part of #742 
- Document it (minimal scope: Add documentation in Usage document)
- Cover basic telemetry

Some update on the design decision after we discussed with @lpetkov :
- There is a restriction on slack about action items which is  you cannot attach them to slack file upload. As a solution, we replied as a thread to file upload, and this reply message is a simple Filter Output input action.
- Once the thread is created, the upcoming commands to that thread is also replied under that thread.

**Comment by mszostok** [comment]
Design:
Don't use dropdowns, use:
![image](https://user-images.githubusercontent.com/17568639/194043359-1c198d48-dd19-4a38-a54f-ce99d7741a38.png)

**Comment by mszostok** [comment]
Attach the filter output section only if message has >= 15 lines.

It can be done in an easy way, pseudocode:
```go
	markdown := interactive.RenderMessage(b.mdFormatter, interactive.Message{})
	lines := strings.SplitN(markdown, "\n", 16)
	if len(lines) == 16 {
		// we reach the limit, let's add a filter section
	}

```

**Comment by pkosiec** [comment]
Update: I will hide the "Delete" option for actionable notifications for now. This doesn't impact the "Actionable messages" feature at all.

I discussed the edge case with @lpetkov (which happens when user explicitly enables the "delete" verb for given K8s resource). Unfortunately I ran out of time - sounds like a rare edge case not worth to invest in right now, as we have different priorities related to Kubecon.

I will create a follow-up task for that and we can see if people really want to delete K8s resources from notifications 🙂
